### ISSUE TITLE
Enhancement: Add XYZ axis labels to the origin axis marker

### ISSUE THREAD

**Issue by jdegenstein** [bernhard-42/three-cad-viewer#11]
See the original issue here: https://github.com/bernhard-42/vscode-ocp-cad-viewer/issues/13

Probably best to have it here where the actual code resides. Here is my non-working attempt to modify `orientation.js` which holds the "trihedron"

```js
    const loader = new THREE.FontLoader();
    loader.load('examples/fonts/helvetiker_regular.typeface.json', function ( font ) {
      var textGeometry = new THREE.TextGeometry( "text", {
        font: font,
        size: 50,
        height: 10,
        curveSegments: 12,
        bevelThickness: 1,
        bevelSize: 1,
        bevelEnabled: true
      });
      var textMaterial = new THREE.MeshPhongMaterial( 
        { color: 0xff0000, specular: 0xffffff }
      );
      const textmesh = new THREE.Mesh(textGeometry, textMaterial);
      this.scene.add(textmesh);
```

**Comment by bernhard-42** [comment]
The examples folder is usually not available in the distribution package of threejs. One needs to somehow add them to the codebase.
Nevertheless, I will check how to add the font to the my viewer and then try to use the orientation/quaternion trick.
Thanks

**Comment by bernhard-42** [comment]
for reference: https://components.ai/docs/typefaces/packages

**Comment by bernhard-42** [comment]
fixed in 1.8.2
